README.md

```diff
@@ -18,6 +18,7 @@ The CLI arguments of `tensorboard` sucks. We can launch it more easily:
 ```
 tb logdir/1 logdir/shell_expansion/*
 tb --port 6006 logdir/*
+tb `gsutil ls "gs://bucket/exp/*"` logdir/*
```

Comment:
Oops, `-d` option is needed for `gsutil`, which only lists matching subdirectory names.

Reply:
Would you want to fix this?